openmc/deplete/chain.py

```diff
@@ -20,7 +20,6 @@
 except ImportError:
     import xml.etree.ElementTree as ET
     _have_lxml = False
-from tqdm import tqdm
```

Comment:
I will miss tqdm.

Reply:
Me too. I really do like tqdm. But, it wasn't really adding all that much for us so I erred on the side of not adding another dependency.